python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -359,9 +466,9 @@ async def check_comp_heartbeat(self, component):
         Parameters
         ----------
         component : `str`
-            Name of the component to follow. Must be one of:
-            atmcs, atptg, ataos, atpneumatics, athexapod, atdome,
-            atdometrajectory
+            Name of the component to follow. The name of the CSC follow the
+            format used to create a `salobj.Remote` (CSCName or CSCName:index,
```

Comment:
Unfortunately this is quite not true. salobj has some support for CSCName:index but Remote's constructor does not.